renovate-base.json5

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      // Enable go indirect dependencies.
```

Comment:
I would have expected direct dependencies... In case we want it for indirect dependencies maybe using `"automerge": false` would be an option to avoid issues.

Reply:
I wasn't sure about including it, but I did it to open the conversation! 😃 

Updating indirect dependencies could help us to solve vulnerabilities earlier (before the direct dependency bumps it).

Additionally, it should not have undefined behaviors unless the indirect dependence has some kind of breaking change (which is not expected as we are only automatically merging minors and patch versions). However, if any minor/patch update introduces a breaking change where the API does not change but the behavior does, then we will face unexpected behaviors.

WDYT?

Reply:
> (which is not expected as we are only automatically merging minors and patch versions)

We are doing some exceptions to this (sarama) and I'm concerned that the same approach could be done by the direct deps maintainers , if that the case we will be jumping that.
IMO this bumps should be done manually (so we can evaluate the risks or perform tests) with the corresponding comment on the `replace` statement, I wouldn't feel comfortable to automate this with a general rule.

Reply:
Fair enough, probably that is the reason why it is disabled by default 🙂

Reply:
I removed the rule to enable indirect dependencies here: [f143c14](https://github.com/newrelic/coreint-automation/pull/13/commits/f143c14b8d0c0d5e32363b05a5bb4189b30b1d07)

Reply:
We can revisit this decision in the future, It is sad not to merge them, but it is true that it could be risky and opening the PR could lead to a excessive noise 😢